src/collects/seashell/backend/project.rkt

```diff
@@ -440,6 +488,22 @@
     [else
       (values #f `#hash((messages . ,messages) (status . "compile-failed")))]))
 
+
+;; (compile-and-run-project/use-runner name tests)
+;; is a wrapper around compile-and-run-project, supplying the file
+;; from the project settings file.
+;; Note: the file arg is used only to determine the question
+(define/contract (compile-and-run-project/use-runner name file tests)
+  (-> project-name? path-string? (listof path-string?)
+      (values boolean?
+              hash?))
+  (define question (car (string-split file "/")))
+  (define file-to-run (get-file-to-run name question))
+  (if (string=? file-to-run "")
```

Comment:
Toss an error here instead, and have the frontend ensure that the runner file is set properly?